packages/smooth_app/lib/views/bottom_sheet_views/user_contribution_view.dart

```diff
@@ -14,72 +13,46 @@ class UserContributionView extends StatelessWidget {
   Widget build(BuildContext context) {
     return Material(
       child: SizedBox(
-        height: MediaQuery.of(context).size.height * 0.9,
-        child: Stack(
+        height: MediaQuery.of(context).size.height * 0.8,
+        child: ListView(
+          shrinkWrap: true,
+          scrollDirection: Axis.vertical,
           children: <Widget>[
-            Column(
-              mainAxisSize: MainAxisSize.max,
-              mainAxisAlignment: MainAxisAlignment.end,
-              children: <Widget>[
-                SizedBox(
-                  height: MediaQuery.of(context).size.height * 0.9,
-                  child: ListView(
-                    shrinkWrap: true,
-                    scrollDirection: Axis.vertical,
-                    children: <Widget>[
-                      Container(
-                        padding: const EdgeInsets.symmetric(horizontal: 20.0),
-                        margin: const EdgeInsets.only(top: 20.0, bottom: 24.0),
-                        child: Text(
-                          AppLocalizations.of(context)!.contribute,
-                          style: Theme.of(context).textTheme.headline1,
-                        ),
-                      ),
-                      Column(
-                        mainAxisSize: MainAxisSize.min,
-                        children: <SmoothListTile>[
-                          //Contribute
-                          SmoothListTile(
-                              text: AppLocalizations.of(context)!
-                                  .contribute_improve_header,
-                              onPressed: () => _contribute(context)),
+            Container(
+              padding: const EdgeInsets.symmetric(horizontal: 20.0),
+              margin: const EdgeInsets.only(top: 20.0, bottom: 24.0),
+              child: Text(
+                AppLocalizations.of(context)!.contribute,
```

Comment:
Instead of evoking multiple times `AppLocalizations.of(context)!` and `Theme.of(context)`, you should use a single variable